src/epcpm/sunspectocsv.py

```diff
@@ -419,17 +439,90 @@ def gen(self) -> typing.List[typing.List[str], int]:
             row.uuid = self.wrapped.uuid
             row.class_name = "DataPointBitfield"
 
-        uses_interface_item = (
-            isinstance(parameter, epyqlib.pm.parametermodel.Parameter)
-            and parameter.uses_interface_item()
-        )
+        rows = []
+        points = list(self.wrapped.children)
+        for child in points:
+            builder = builders.wrap(
+                wrapped=child,
+                model_type=self.model_type,
+                parameter_uuid_finder=self.parameter_uuid_finder,
+                model_id=self.model_id,
+                model_offset=self.model_offset,
+                address_offset=self.address_offset,
+            )
+            child_row = builder.gen()
+            rows.append(child_row)
+
+        # Insert the parent DataPointBitfield ahead of the list of DataPointBitfieldMember rows.
+        rows.insert(0, row)
```

Comment:
replace 0 with something like START_OF_LIST_INDEX ?

Reply:
Just my 2 cents but I am cool with 0's since that's generally a given for start of array/list index.

Reply:
While the insert(0, x) is a common way to insert at the front of a list, there is a little smell here. I didn't have to wait until the end to add the first row. I moved the add prior to the rest of the rows being added. See fc8890ed79698211e5fc27103b241f25a4ba60c9